tao/WORKSPACE

```diff
@@ -0,0 +1,41 @@
+workspace(name = "org_tao_bridge")
+
+# Initialize the tao_bridge repository and all dependencies.
+#
+# The cascade of load() statements and workspace?() calls works around the
+# restriction that load() statements need to be at the top of .bzl files.
+# E.g. we can not retrieve a new repository with http_archive and then load()
+# a macro from that repository in the same file.
+
+load("@//:workspace2.bzl", "tao_bridge_workspace2")
+
+tao_bridge_workspace2()
+
+load("@//:workspace1.bzl", "tao_bridge_workspace1")
+
+tao_bridge_workspace1()
+
+load("@//:workspace0.bzl", "tao_bridge_workspace0")
+
+tao_bridge_workspace0()
+
+load("@local_config_tf_protobuf//:tf_protobuf_workspace.bzl", "tf_protobuf_workspace")
+
+tf_protobuf_workspace()
+
+# Initialize TensorFlow's external dependencies.
+load("@org_tensorflow//tensorflow:workspace3.bzl", "tf_workspace3")
```

Comment:
We have specifically used `com_google_protobuf` `com_google_googletest` `zlib` `cuda` `rocm` configs. But it is important to load tf's workspace for using the same `BUILD` file for both `tao_bridge` and `tao_compiler` build, since the `BUILD` will load plenty things from `org_tensorflow`. Besides, the workspace files should all be loaded for it to work.